workflow-job

```diff
@@ -508,7 +508,6 @@ flows["${sha1}"] = {
 		    }
 
                     if (packagageOnGitlab) {
-                        def gitlabToken = gitlabToken();
                         sh """curl
 			   |-X POST
 			   |-Ftoken=${gitlabToken}
```

Comment:
this variable is used here. Are you sure it will call the function instead?

Reply:
Yes, the use in workflow is to have the result of function call stored in a variable.

Exemple: dockerUser, dockerPassword...